.github/workflows/build_docs.yml

```diff
@@ -65,7 +65,7 @@ jobs:
           git config --local user.name "GitHub Action"
           git commit -m "Update develop documentation"
 
-      - uses: ad-m/github-push-action@v0.6.0
+      - uses: ad-m/github-push-action@v0.6.1
```

Comment:
I'm guessing this action got caught in then "find/replace" crossfire? Should be free to revert!